src/FastState.cpp

```diff
@@ -69,51 +69,30 @@ bool FastState::is_move_legal(int color, int vertex) {
                 !board.is_suicide(vertex, color));
 }
 
-void FastState::play_pass(int color) {
-    m_movenum++;
-
-    m_lastmove = FastBoard::PASS;
+void FastState::play_move(int vertex) {
+    play_move(board.m_tomove, vertex);
+}
 
+void FastState::play_move(int color, int vertex) {
```

Comment:
This diff isn't the best, I'd suggest comparing this by hand with the code that used to start at line 97